### PR TITLE
[15.0][MIG] hr_employee_calendar_planning

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
Work done:
- Fix tests
- Fix init_hook
- re-generate README.rst
- Fix odoo-pylint warnings

### CODE REVIEW

**Comment by victoralmau** [comment]
Please, cherry pick https://github.com/OCA/hr/pull/1094

**Comment by victoralmau** [comment]
Please, cherry pick https://github.com/OCA/hr/pull/1095#issuecomment-1110021233

**Review by victoralmau — CHANGES_REQUESTED** [review]
Please, don't remove tests, It is very important to validate all use cases.

**Comment by pegonzalezspesol** [comment]
> Please, don't remove tests, It is very important to validate all use cases.

@victoralmau PR rebased with latests 14.0 status. Can you please have a look?

**Review by victoralmau — APPROVED** [review]
Code review OK.

File: hr_employee_calendar_planning/models/resource_calendar.py [hunk 1]
```diff
@@ -20,8 +20,12 @@ def _check_active(self):
             )
             if total_items:
                 raise ValidationError(
-                    _("%s is used in %s employee(s). You should change them first.")
-                    % (item.name, total_items)
+                    _(
+                        "%(item_name)s is used in %(total_items)s employee(s)."
```
Comment (pedrobaeza) [inline hr_employee_calendar_planning/models/resource_calendar.py:24]: Can you update at least Spanish translation?

File: hr_employee_calendar_planning/tests/test_hr_employee_calendar_planning.py [hunk 2]
```diff
@@ -8,10 +8,10 @@
 from ..hooks import post_init_hook
 
 
-class TestHrEmployeeCalendarPlanning(common.SavepointCase):
+class TestHrEmployeeCalendarPlanning(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
-        super().setUpClass()
+        super(TestHrEmployeeCalendarPlanning, cls).setUpClass()
```
Comment (pedrobaeza) [inline hr_employee_calendar_planning/tests/test_hr_employee_calendar_planning.py:14]: No need of this diff. It was OK before

**Comment by victoralmau** [comment]
Can you apply the latest changes to merge it?

**Comment by victoralmau** [comment]
Please, cherry-pick https://github.com/OCA/hr/pull/1128

**Comment by victoralmau** [comment]
Superseed by https://github.com/OCA/hr/pull/1138

**Pull request closed without merging**